morph.go

```diff
@@ -353,6 +354,20 @@ func execDeploy(hosts []nix.Host) (string, error) {
 		}
 
 		if deployReboot {
+			var (
```

Comment:
May I suggest extracting this entire `deployReboot`section to a separate functions, e.g. `deployReboot()`, since `execDeploy()` is becoming really big.

Reply:
Sure, can do, totally reasonable.

Reply:
Most of the current `execFoo` methods take a list of hosts to work on, so this could also be implementer as a method on the host interface.

Reply:
If it's too much of a pain to do, then don't. I just thought it would make `execDeploy` more readable. :)

Reply:
It only depends on the SSHContext, so it's really easy to move around.